src/api/keymap/db/mediacontrols.js

```diff
@@ -70,6 +70,22 @@ const MediaControlTable = {
         primary: "Vol-",
         verbose: "Volume down"
       }
+    },
+    {
+      code: 23663,
+      labels: {
+        top: "Media",
+        primary: "Brt+",
+        verbose: "Brithness up"
```

Comment:
Brightness, rather than Brithness, throughout.

Reply:
Here comes the light :-)